### ISSUE TITLE
custom-config.sh fails due to lack of eth2 interface

### ISSUE THREAD

**Issue by chen23** [F5Networks/f5-aws-cloudformation#62]
## Description
Unable to complete onboarding using: https://github.com/F5Networks/f5-aws-cloudformation/blob/ada5cd3a9d78ce40f404e020edf9245a22a30e30/supported/failover/across-net/via-api/3nic/existing-stack/payg/f5-existing-stack-across-az-cluster-payg-3nic-bigip.template

The script expects "eth2", but no device exists.

output from ifconfig:

```
eth0      Link encap:Ethernet  HWaddr 06:B2:8F:E7:BD:D0
          inet6 addr: fe80::4b2:8fff:fee7:bdd0/64 Scope:Link
          UP BROADCAST RUNNING ALLMULTI MULTICAST  MTU:9001  Metric:1
          RX packets:36699 errors:0 dropped:0 overruns:0 frame:0
          TX packets:8207 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000
          RX bytes:50455099 (48.1 MiB)  TX bytes:3389568 (3.2 MiB)

external  Link encap:Ethernet  HWaddr 06:15:8F:F6:A7:EA
          inet addr:10.1.12.129  Bcast:10.1.12.255  Mask:255.255.255.0
          inet6 addr: fe80::415:8fff:fef6:a7ea/64 Scope:Link
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:2403 errors:0 dropped:0 overruns:0 frame:0
          TX packets:2965 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1
          RX bytes:603102 (588.9 KiB)  TX bytes:465563 (454.6 KiB)

internal  Link encap:Ethernet  HWaddr 06:72:52:AC:67:92
          inet addr:10.1.22.44  Bcast:10.1.22.255  Mask:255.255.255.0
          inet6 addr: fe80::472:52ff:feac:6792/64 Scope:Link
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
          TX packets:4 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1
          RX bytes:0 (0.0 b)  TX bytes:400 (400.0 b)

lo        Link encap:Local Loopback
          inet addr:127.0.0.1  Mask:255.255.255.0
          inet6 addr: ::1/128 Scope:Host
          UP LOOPBACK RUNNING  MTU:65536  Metric:1
          RX packets:41591 errors:0 dropped:0 overruns:0 frame:0
          TX packets:41591 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1
          RX bytes:10398451 (9.9 MiB)  TX bytes:10398451 (9.9 MiB)

lo:1      Link encap:Local Loopback
          inet addr:127.2.0.2  Mask:255.255.255.0
          UP LOOPBACK RUNNING  MTU:65536  Metric:1

mgmt      Link encap:Ethernet  HWaddr 06:B2:8F:E7:BD:D0
          inet addr:10.1.2.214  Bcast:10.1.2.255  Mask:255.255.255.0
          inet6 addr: fe80::4b2:8fff:fee7:bdd0/64 Scope:Link
          UP BROADCAST RUNNING MULTICAST  MTU:9001  Metric:1
          RX packets:7314 errors:0 dropped:0 overruns:0 frame:0
          TX packets:6979 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000
          RX bytes:48402487 (46.1 MiB)  TX bytes:3323880 (3.1 MiB)

tmm       Link encap:Ethernet  HWaddr 00:98:76:54:32:10
          inet addr:127.1.1.254  Bcast:127.1.1.255  Mask:255.255.255.0
          inet6 addr: fe80::298:76ff:fe54:3210/64 Scope:Link
          UP BROADCAST RUNNING MULTICAST  MTU:9282  Metric:1
          RX packets:14416 errors:0 dropped:0 overruns:0 frame:0
          TX packets:22608 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1
          RX bytes:809126 (790.1 KiB)  TX bytes:27402898 (26.1 MiB)

tmm_bp    Link encap:Ethernet  HWaddr 02:01:23:45:00:FE
          inet addr:127.20.0.254  Bcast:127.20.255.255  Mask:255.255.0.0
          inet6 addr: fe80::1:23ff:fe45:fe/64 Scope:Link
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
          TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1
          RX bytes:0 (0.0 b)  TX bytes:0 (0.0 b)
```
custom-config.sh:
```
...
GATEWAY_MAC2=`ifconfig eth2 | egrep HWaddr | awk '{print tolower($5)}'`
...
```
results in error in logs like:

```
2018-12-19T14:27:35.337Z error: [pid: 17446] [scripts/runScript.js] Syntax Error: Invalid IP address: "...1"
```

emulating the script:
```
[admin@ip-10-1-2-214:Active:Disconnected (Trust Domain Only)] ~ # ifconfig eth2
eth2: error fetching interface information: Device not found
```

changing to "eth2" to "internal" seems to get the original behavior.

```
Sys::Version
Main Package
  Product     BIG-IP
  Version     13.1.1
  Build       0.0.4
  Edition     Final
  Date        Fri Jul 20 17:55:49 PDT 2018
```
## Template
For bugs, enter the template with which you are experiencing issues below.

https://raw.githubusercontent.com/F5Networks/f5-aws-cloudformation/master/supported/failover/across-net/via-api/3nic/existing-stack/payg/f5-existing-stack-across-az-cluster-payg-3nic-bigip.template

## Severity Level
For bugs, enter the bug severity level. Do not set any labels.

Severity: 2

**Comment by edwoodjrjr** [comment]
Tracking this in internal issue #854.

**Comment by chen23** [comment]
weird.  it was in us-east-1, N Virginia.  us-east-1d/e

**Comment by edwoodjrjr** [comment]
It is weird, although I saw a similar transient issue in Azure the other day that IMO was just the SDN not being ready yet. 

I've run two successful tests in the same region, plus a couple more tests of the 3 nic standalone, which uses the same script.  I'll leave this open for now.  Let us know if it comes up again, or if we can close this issue, before the next release.

**Comment by edwoodjrjr** [comment]
Hi Eric, did you ever see this again?

**Comment by crosbygw** [comment]
Look like an issue with command ordering; ie vlan creation in network.js is renaming interfaces from et1/eth2 to external and internal. Adding notes to internal issue noted above.

**Comment by crosbygw** [comment]
Issue resolved in release v4.1.2
